Remant/Deep - Conversation Option and Remnant Awareness (#5111)
endless-sky/endless-sky: data/remnant/remnant missions.txt

mission "Remnant: Follow-up on Terminus Researchers A"
	name "Check up on Wormhole Researchers"
	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
	minor
	source

> 		government Remnant
> 	destination Bounty

Reviewer: ```suggestion
		government "Remnant"
	destination "Bounty"
```

> 	to offer

Reviewer: These current conditions make it such that you could potentially get this mission to offer while you are researching the Ember Waste for the Deep scientists. You should probably add `not "Deep: Remnant 0: offered"`.

Author: Done. Thanks!

Author: OK, so this looks like it is now done

		has "Remnant: Learning Sign 2: done"
		has "Terminus exploration: done"

> 		not "Deep: Wormhole Research: done"
> 		not "Remnant: Follow-up on Terminus Researchers B: done"
> 		not "Remnant: Follow-up on Terminus Researchers B: active"

Reviewer: ```suggestion
		not "Deep: Remnant 0: offered"
```

		random < 40
	on offer
		conversation

> 			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`

Reviewer: Given that the player is more likely than not to have not thought about being a pirate, I wouldn't have this line here implying that they did.

Author: OK. Removed.

Author: Given that capturing pirate ships is basically being a privateer (aka legalized piracy), it isn't that far of a stretch. There's not a big leap between capturing pirates and capturing merchants. James even talks about capturing ships in the intro.

But either way, removed that line for now.

> 			branch secrets

Reviewer: It seems to me like this branch should not exist; the current assumption of this mission is that you never followed up with the researchers, only ever retrieving the drone. (See the fact that this mission has a destination on Bounty.)

				has "deep: did not reveal remnant"

> 			`	Thinking back, it suddenly occurs to you that you helped retrieve a science drone for some researchers on Bounty - researchers who were studying the wormhole in Terminus. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes. It would only be a matter of time until someone figures out how to use the key stones - either by luck or by research, and then the Remnant's secrecy would vanish.`

Reviewer: Research research research research research research research.
Research research research.
Research research.

				goto terminuschoice
			label secrets

> 			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`

Reviewer: The above, but with one extra "research" per line.

			label terminuschoice
			choice
				`	(Find a prefect.)`
				`	(Forget about it.)`
					decline

> 			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`

Reviewer: ```suggestion
			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
```

> 			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`
> 			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`

Reviewer: ```suggestion
			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
```

			branch confirmation
				has "deep: did not reveal remnant"

> 			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`

Reviewer: ```suggestion
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
```

> 			goto investigatechoice

Reviewer: There is no `label investigatechoice`.

			label confirmation

> 			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`

Reviewer: ```suggestion
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`
```

			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
			choice
				`	"I'd be happy to."`
				`	"I'd prefer not to."`
					decline
			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
				accept
	on complete

> 		conversation

Reviewer: Why have this conversation here if the next mission is a landing one? Just put this conversation in the next mission's on offer.

> 			`Arriving on Bounty, you take a discreet look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`

Reviewer: If you knew Ivan by name then you'd already know they're in the Deep, not on Bounty, so why ever come here? I also changed the source of the Terminus Exploration mission to Deep space, so even then you know that these researchers are from the Deep, not from Bounty.

Author: OK, that'll be adjusted a bit, then.

mission "Remnant: Follow-up on Terminus Researchers 2A"
	name "Check up on Terminus Researchers"
	description "Tracking down the wormhole researchers."

> 	source Bounty
> 	destination
> 		government Deep

Reviewer: ```suggestion
	source "Bounty"
	destination
		government "Deep"
```

	to offer
		has "Remnant: Follow-up on Terminus Researchers: done"
	on offer
		conversation

> 			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. A quick check of the directory lists a number of planets where they could be found.`

Reviewer: This seems like unnecessary padding to me. Why couldn't I just look up the university's location of I know what it is?

			choice
				`	(Stop investigating.)`
					decline
				`	(Continue the investigation)`

> 				accept

Reviewer: ```suggestion
					accept
```

	on complete

> 		conversation

Reviewer: Again with the on complete convo that should be in the next mission's on offer.

> 			`	Once you've landed on <planet> you look up the local university and pay it a visit. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`

Reviewer: ```suggestion
			`Once you've landed on <planet> you look up the local university and pay it a visit. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`
```

mission "Remnant: Follow-up on Terminus Researchers 3A"
	name "Check up on Terminus Researchers"

> 	description "Returning with information on the wormhole researchers"

Reviewer: ```suggestion
	description "Returning with information on the wormhole researchers."
```

	source

> 		government Deep
> 	destination
> 		government Remnant

Reviewer: ```suggestion
		government "Deep"
	destination
		government "Remnant"
```

	to offer
		or
			has "Remnant: Follow-up on Terminus Researchers 2: done"
			has "Remnant: Follow-up on Terminus Researchers B: done"
	on offer
		conversation

> 			`Back on board <ship>, you take a moment to flip through the papers.  You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`

Reviewer: ```suggestion
			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
```

> 			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: They have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`

Reviewer: ```suggestion
			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: they have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
```

			`	Prefect Torza is definitely going to be interested in seeing this.`

> 			accept

Reviewer: ```suggestion
				accept
```

	on complete
		payment 2000000
		conversation
			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fail.`

> 			`	Torza seems grimly optimistic on receiving your report "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time, and have proven yourself loyal to us. So we are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide."`

Reviewer: ```suggestion
			`	Torza seems grimly optimistic on receiving your report. "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time, and have proven yourself loyal to us. So we are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide.`
```

			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`

# The B mission is for players that kept the Remnant secret from the Deep, but did the wormhole research.
mission "Remnant: Follow-up on Terminus Researchers B"
	name "Check up on Wormhole Researchers"

> 	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."

Reviewer: As mentioned before, if you know that the researchers are in the Deep, why would you go to Bounty?

	minor
	source

> 		government Remnant

Reviewer: ```suggestion
		government "Remnant"
```

	destination "Valhalla"
	to offer
		has "Remnant: Learning Sign 2: done"

> 		has "Deep: Wormhole Research: done"

Reviewer: This mission doesn't complete. Therefore, you need to check if it has offered.
```suggestion
		has "Deep: Wormhole Research: offered"
```

		not "Remnant: Follow-up on Terminus Researchers A: done"

> 		not "Remnant: Follow-up on Terminus Researchers B: active"

Reviewer: These conditions are unnecessary as the other conditions of both this set and the prior set are mutually exclusive; you can not both not have done the wormhole research and have done it at the same time.

		random < 40
	on offer

> 		conversation

Reviewer: Given that the conversations for these two missions repeat a lot of the start, you could just make them the same conversation with a branch based off of which mission you're currently on.

			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
			choice
				`	(Find a prefect.)`
				`	(Forget about it.)`
					decline

> 			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`

Reviewer: ```suggestion
			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
```

			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`
			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`

> 			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers studying the wormhole. He appreciates that you kept them focused on the wormhole train instead of revealing the Remnant, but still appears concerned.`

Reviewer: ```suggestion
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers studying the wormhole. He appreciates that you kept them focused on the wormhole train instead of revealing the Remnant, but still appears concerned.`
```

			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
			choice
				`	"I'd be happy to."`
				`	"I'd prefer not to."`
					decline
			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
				accept
	on complete
		conversation

> 			`	Once you've landed on <planet> you look up the university that Ivan worked for. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here at their university. They proudly give you copies and wish you good luck in your studies.`

Reviewer: ```suggestion
			`Once you've landed on <planet> you look up the university that Ivan worked for. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here at their university. They proudly give you copies and wish you good luck in your studies.`
```

				goto terminuschoice
			label secrets
			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
			label terminuschoice
			choice
				`	(Find a prefect.)`
				`	(Forget about it.)`
					decline
			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`

> 			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`

Reviewer 2: ```suggestion
			`	"Greetings Captain. I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
```
OR
```suggestion
			`	"Greetings Captain; I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant and gestures for you to do the same.`
```

			branch confirmation
				has "deep: did not reveal remnant"
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
			goto investigatechoice
			label confirmation
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`

> 			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`

Reviewer 2: ```suggestion
			`	Thinking carefully, he says, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
```
Obviously `asks` _can_ work here, but it feels weird knowing the following sentence is a statement instead of question.

			choice
				`	"I'd be happy to."`
				`	"I'd prefer not to."`
					decline
			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
				accept
	on complete
		conversation
			`Arriving on Bounty, you take a discreet look around the spaceport, but you don't find the researchers or any sign of them. It appears that they packed up and left since the last time you saw them. Time to look around for some info.`

mission "Remnant: Follow-up on Terminus Researchers A-2"
	name "Check up on Terminus Researchers"

> 	description "Tracking down the wormhole researchers."

Reviewer 2: This might need to be more informative about what to do to complete the mission.

	source "Bounty"
	destination
		government "Deep"
	to offer
		has "Remnant: Follow-up on Terminus Researchers A-1: done"
	on offer
		conversation
			`A few hours of searching later, you find a lead: It seems that launching probes into the Terminus system isn't exactly a common activity, so their permits stand out from the rest. A helpful clerk in the spaceport office pulled up their application and found that they are listed as part of a university in the Deep, although it doesn't mention which planet their specific campus is on. A quick check of the directory lists a number of planets where they could be found.`
			choice
				`	(Stop investigating.)`
					decline
				`	(Continue the investigation)`
					accept
	on complete
		conversation

> 			`Once you've landed on <planet> you look up the local university and pay it a visit. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`

Reviewer 2: ```suggestion
			`Once you've landed on <planet>, you look up the local university and pay it a visit. After thinking about it for a minute, you decide that the best source of information would probably be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief searching, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`
```

mission "Remnant: Follow-up on Terminus Researchers AB-3"
	name "Check up on Terminus Researchers"

> 	description "Returning with information on the wormhole researchers."

Reviewer 2: Same here: more information about what to do.

	source
		government "Deep"
	destination
		government "Remnant"
	to offer
		or
			has "Remnant: Follow-up on Terminus Researchers A-2: done"
			has "Remnant: Follow-up on Terminus Researchers B: done"
	on offer
		conversation
			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
			`	All the academic stuff aside, the summary on the most recent paper confirms that the researchers are making progress: they have determined that the spatial anomaly in the Terminus system is in fact a wormhole with some kind of instability. They also speculate that it may be possible for a ship to artificially stabilize the wormhole enough for it to pass through unharmed.`
			`	Prefect Torza is definitely going to be interested in seeing this.`
				accept
	on complete
		payment 2000000
		conversation

> 			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fail.`

Reviewer 2: ```suggestion
			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation is about to fall apart.`
```
OR
```suggestion
			`The trip back to the Ember Wastes is long, and it gave you plenty of opportunities to think about what this research could lead to. You also wondered about what the Remnant will do, as it seems obvious that their current isolation plan is about to fail.`
```
Nothing is grammatically incorrect, but I thought the usage of `fail` felt weird without a little rewording. All optional.

> 			`	Torza seems grimly optimistic on receiving your report. "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time, and have proven yourself loyal to us. So we are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide.`

Reviewer 2: ```suggestion
			`	Torza seems grimly optimistic on receiving your report. "If this had happened without us being aware, it could have gone badly. Fortunately for us, you were in the right place at the right time and have proven yourself loyal to us. We are now forewarned and can use this time to plan and prepare for it. Perhaps re-establishing contact with the rest of humanity could even be to our benefit, but that will be for the council to decide.`
```

			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`

# The B mission is for players that kept the Remnant secret from the Deep, but did the wormhole research.
mission "Remnant: Follow-up on Terminus Researchers B"
	name "Check up on Wormhole Researchers"
	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
	minor
	source
		government "Remnant"
	destination "Valhalla"
	to offer
		has "Remnant: Learning Sign 2: done"
		has "Deep: Wormhole Research: offered"
		not "Remnant: Follow-up on Terminus Researchers A: done"
		not "Remnant: Follow-up on Terminus Researchers B: active"

> 		random < 40

Reviewer 2: This could be a little low considering that there's possibly relevant story to be read here. Just my opinion.

	on offer
		conversation

> 			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`

Reviewer 2: ```suggestion
			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you first got your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants (and on your darker days - contemplating being a pirate).`
```

> 			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`

Reviewer 2: ```suggestion
			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to new vessels venturing into the Ember Wastes.`
```
Suggested to minimize usage of `research`.

			choice
				`	(Find a prefect.)`
				`	(Forget about it.)`
					decline

> 			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`

Reviewer 2: ```suggestion
			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons then give you directions down a hallway. You quickly find yourself in an armory where several Remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
```

> 			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`

Reviewer 2: ```suggestion
			`	"Greetings Captain. I am Prefect Torza. I was told you had some information that could be important?"`
```
OR
```suggestion
			`	"Greetings Captain; I am Prefect Torza. I was told you had some information that could be important?"`
```

> 			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`

Reviewer 2: ```suggestion
			`	He takes a seat on a bench on the far side of the room from the other Remnant and gestures for you to do the same.`
```

			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers studying the wormhole. He appreciates that you kept them focused on the wormhole train instead of revealing the Remnant, but still appears concerned.`

> 			`	Thinking carefully, he asks, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`

Reviewer 2: ```suggestion
			`	Thinking carefully, he says, "It is clear that we need to keep an eye on this situation. You know more about the rest of the galaxy than we do, so could you find them and see how much progress they have made?"`
```
Same comment as suggested in similar line earlier.

			choice
				`	"I'd be happy to."`
				`	"I'd prefer not to."`
					decline
			`	"Excellent. See what you can find out about their research and results, but it would probably be wise to avoid making direct contact with them."`
				accept
	on complete
		conversation

> 			`Once you've landed on <planet> you look up the university that Ivan worked for. After thinking about it for a minute, you decide that probably the best source of information would be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here at their university. They proudly give you copies and wish you good luck in your studies.`

Reviewer 2: ```suggestion
			`Once you've landed on <planet>, you look up the university that Ivan worked for. After thinking about it for a minute, you decide that the best source of information would probably be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief research, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here at their university. They proudly give you copies and wish you good luck in your studies.`
```

			choice
				`	(Stop investigating.)`
					decline
				`	(Continue the investigation)`
					accept
	on complete
		conversation
			`Once you've landed on <planet>, you look up the local university and pay it a visit. After thinking about it for a minute, you decide that the best source of information would probably be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief searching, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`

mission "Remnant: Follow-up on Terminus Researchers AB-3"
	name "Check up on Terminus Researchers"
	description "Returning with information on the wormhole researchers."
	source
		government "Deep"
	destination
		government "Remnant"
	to offer
		or
			has "Remnant: Follow-up on Terminus Researchers A-2: done"
			has "Remnant: Follow-up on Terminus Researchers B: done"
	on offer
		conversation

> 			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the summary is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`

Reviewer 3: ```suggestion
			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the abstract is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`
```

			
			`	You approach a spaceport information desk and ask to see Prefect Torza, who you spoke to the last time you informed the Remnant about the Deep researching the wormhole. Luckily, Torza is on-world at the moment, and you are told to meet him in the cafeteria. When you arrive, you find that Torza has just finished a meal and was leaving.`
			`	"Ah, Captain. I was told you were looking for me. We should find a private place to talk."`
			`	You follow Torza to a similar armory as before. "Is there an update to the research on the Threshold that the Deep are doing?" he asks after sitting down.`
			choice
				`	"I've helped the scientists learn how to pass through the wormhole."`
				`	"The scientists have discovered how to pass through the wormhole."`
					goto discovered
			
			apply
				set "remnant: admitted to helping deep"
			`	Torza's eyes go wide. "Why would you do this?" he asks. "Do you not understand that our secrecy is of upmost importance?"`
			choice
				`	"I don't believe that they will be a threat to you."`
					goto threat
				`	"I'm sorry, but I helped them before coming to understand this."`
			
			apply
				set "remnant: lied to torza: understanding"

> 			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`

Reviewer: This conversation isn't occurring through sign language. Verbiage further up in this conversations is "says," not "signs."

> 			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`

Reviewer 2: Ah, I see. I still personally think using `yell` is iffy given the Remnant sing to speak instead of using normal inflexion, but it doesn't matter all that much if you all would prefer not to change it.

> 			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`

Author: It might not be ideal, but I can't think of a good alternative, and it does fit the situation. As such, I'm fine with it.

> 			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`

Reviewer: Well I don’t see why “yelling” from the Remnant can’t be like a hefty opera-like singing.

> 			`	"Liar!" Torza yells at you while standing up from the bench. "You told us of these scientists after we had already trusted you with a license." Torza walks away from you angrily, then turns his head.`

Author: Sure, that's how I picture it.

			choice
				`	"I promise."`
					goto digitize
				`	"I'll try."`
			`	"Funny," Ivan says with a blank expression.`
			label digitize
			`	"Let me just get these folders digitized for you, then I can hand you a data chip with everything on it. It'll only be a few minutes."`
			apply
				set "remnant: got deep anomaly research"
			`	Ivan leaves the room with all the folders, and when he returns he hands you a data chip before shoving all the folders back into the filing cabinet. "Well, anything else you need?"`
			`	"No. That's all I was here for," you respond. "Good luck on the rest of your work."`
			`	"Thank you," Ivan says. "We might be in touch in the future. I might need your help again for this project, hopefully during a less dire situation than last time."`
				decline
			
			label "retrieved before"
			`	You return to the office complex where Ivan's office is and ask the receptionist if Ivan is available, and he looks at you with confusion.`
				goto moved
			
			label gone
			`	You ask a receptionist what Ivan's room number is, and he looks at you with confusion.`
			
			label moved

> 			`	"Ivan Haataja? He moved out weeks ago. Said something about 'moving up in this world' while dragging a pallet of filing cabinets out of the building. I can look up where he is now if you need me to."`

Author: Technically, yes, a pallet is the wooden structure that gets loaded up. This expression is roughly equivalent to saying "a wheelbarrel full" or "a truck-full." None of them are technically units of measurement, but they still convey a sense of quantity.

> 			`	"Ivan Haataja? He moved out weeks ago. Said something about 'moving up in this world' while dragging a pallet of filing cabinets out of the building. I can look up where he is now if you need me to."`

Reviewer: I wouldn't exactly describe a pallet as "full."

> 			`	"Ivan Haataja? He moved out weeks ago. Said something about 'moving up in this world' while dragging a pallet of filing cabinets out of the building. I can look up where he is now if you need me to."`

Reviewer 2: Sounds good. Feel free to resolve.

			`	You approach one of the receptionist desks and ask to see Ivan Haataja. "It looks like he's available right now," she says. "We'll let him know you're coming and have someone here to escort you to Doctor Haataja's office shortly."`
			`	You're escorted through what seems like endless hallways and stairwells, the entire building being as impressively decorated as the lobby, before finally coming to Ivan's office. You thank the escort for bringing you here (as you would have been lost for hours without him, surely why they have people just for escorting guests) and knock on Ivan's door.`
			`	Ivan opens to door with a big smile on his face. "Captain <last>! So great to see you here. Please, have a seat." Ivan ushers you inside his office, an impressively large room with a window wall overseeing a beautiful courtyard. You sit down in a cushioned chair on the opposite side a desk. You lean back into the chair and get comfortable. "I know the chairs are comfortable," Ivan says. "Don't fall asleep on me though."`

> 			branch second

Reviewer: Well there ARE differences, but your suggestion would remove them. :)

> 			branch second

Reviewer 2: Whoops! I entirely missed that. I've edited the suggestion to reflect it, but it looks like Amazinite already beat me to the punch.

				"retrieved information" == 2
			`	Ivan then takes a seat in his own cushioned chair. "Welcome to my office. This is miles above what I use to have. After gaining the attention of the Deep government with the discovers we were making, they immediately had my office moved from the dingy complex it use to be at to here. Much of that work I have you the thank for.`
				goto reason
			
			label second

> 			`	Ivan then takes a seat in his own cushioned chair. "Welcome to my office. Much better than the old place, isn't it? After gaining the attention of the Deep government with the discovers we were making, they immediately had my office moved from the dingy complex it use to be at to here. Much of that work I have you the thank for.`

Reviewer: The difference in the second sentence is intentional.
```suggestion
			`	Ivan then takes a seat in his own cushioned chair. "Welcome to my office. Much better than the old place, isn't it? After gaining the attention of the Deep government with the discoveries we were making, they immediately had my office moved from the dingy complex it used to be at to here. I have you to thank for much of that work.`
```

			`	Ivan turns back to you. "As for those alien creatures, they're similarly complexing. Humanity has found life on other worlds before, but it has always conformed to our ideas of what life is given our experience on Earth centuries ago. But these creatures are something else entirely. There are no terrestrial planets in the system you found them in suitable for life, so they must have evolved on the gas giants in the system. How life can even begin in such an environment, let alone evolve to be able to travel the vacuum of space, is a mystery to us.`
			`	"By the way, do you want to take the honors of naming these creatures? My team has some ideas, but we're not particularly convinced by any of them, so I thought I'd ask you."`
			choice
				`	"How about 'void sprites'?"`
					goto sprites
				`	"How about 'Floaty McFloat Face'?"`
					goto joke
				`	"Sorry, I don't have any ideas."`
			

> 			`	"That's unfortunate," Ivan says. "I've been thinking of something in relation to the void of space. I'll come up with something eventually."`

Reviewer 2: Don't like the double `something`, but can't think of anything better myself.

Reviewer: Yeah, I wasn't a fan of it either but I moved past it instead of getting myself stuck here.
```suggestion
			`	"That's unfortunate," Ivan says. "I've been thinking of a name in relation to the void of space. I'll come up with something eventually."`
```

			
			label next
			apply
				set "remnant: got deep wormhole and sprite research"
			`	He presses a button on the desk. "Abigail, could you please make a copy of the encrypted folders 'TAB-RP,' 'WLT-RP', and 'GGA-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
				goto reencrypt
			
			label remnant
			apply
				"ivan's suspicion" = ( "retrieved information" / 2 * "deep: knew of stones before scientists" ) + "deep: knew of stones before scientists" + 2 * "Deep: Remnant: Engines: declined" + 2 * "Deep: Remnant: Engines: failed" + "Deep: Remnant Surveillance: declined" + "Deep: Remnant Surveillance: failed"
			
			branch technology
				has "remnant: told that deep has technology"
			
			branch surveillance
				has "remnant: told about surveillance"
			
			choice
				`	"I was wondering what your research on those Remnant ship scans has brought up."`
				`	"Could I get a copy of your research on those Remnant ship scans?"`
					goto suspicion
			branch cubes
				has "Deep: Remnant Surveillance: done"
			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`

> 			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but the hull of these ships have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now when it comes to ships designed by say Betelgeuse or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`

Reviewer 2: ```suggestion
			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but their hulls have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now, when it comes to ships designed by, say, Betelgeuse, or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`
```
Removed the double `the(se) ships`, and added some commas. It looks a little crowded with that many commas, but it's grammatically correct, and I like to think it helps the reader imagine the rhythm someone would actually speak the phrase aloud in.

> 			`	"Aside from the strange hull structure, these ships also seem to have a considerably amount of equipment integrated directly into the ships themselves. In-built scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of in-built device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`

Reviewer 2: ```suggestion
			`	"Aside from the strange hull structure, they also seem to have a considerable amount of equipment integrated directly into the ships themselves. Built-in scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of built-in device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`
```
Did a little googling since I rarely hear `in-built`, and it seems to be a primarily British synonym to "built-in", so I'd suggest using "built-in" instead. Besides that, just a little fix and removal of another double `the(se) ships`.

			`	"Do you think I could get a copy of your research?" you ask.`
				goto suspicion
			
			label technology
			choice
				`	"I was wondering what your research on Remnant technology has brought up."`
				`	"Could I get a copy of your research on Remnant technology?"`
					goto suspicion
			branch cubes
				has "Deep: Remnant Surveillance: done"
			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`

> 			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size, more so than even atomic engines of a similar size, while not producing very much heat at all. Their major downside though seems to be the immense amount of energy they draw. The energy drain is not surpising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is somethign spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`

Reviewer 2: ```suggestion
			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size, more so than even atomic engines of a similar size, while not producing very much heat at all. Their major downside though seems to be the immense amount of energy they draw. The energy drain is not surprising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is something spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`
```

			
			branch skip
				not "Deep: Remnant: Generators: done"

> 			`	He pulls out a schematic showing the generator. "The generator on the other hand wasn't all too interesting. Little more than an improved generator of the type you might find down in the Rim. Somewhat interesting that it doesn't produce much energy when compared to something like a fusion reactor given the energy requirement of their engines, but I suppose this people are concerned more with keeping their ships running cool then having excess energy. The one interesting characteristic we discovered though is that the generator seemed to handle the effects of ionization, like something the Hai ion cannons apply, much better than normal. It seemed to be an intentional result of how the generator was designed."`

Reviewer 2: Is there a check earlier that the player has met the Hai? I haven't seen one, but I'm only looking at the new stuff so it may have been earlier.

Reviewer: The player must have met the Hai to even get to this point as a function of the offer requirements for previous missions.

Reviewer 2: Sweet, just wanted to check.

			
			branch skip
				not "Deep: Remnant: Inhibitor Cannon: done"

> 			`	He then pulls out a schematic showing the weapon. "And finally the weapon you brought us. The Inhibitor Cannon is quite the technical marvel, having properties we've never seen before. Imagine a Particle Cannon, but half the size and with more range. Two Inhibitor Cannons would take up about the same amount of space as a single Particle Cannon, yet the Inhibitor Cannons would deal more damage. On top of that, the projectiles of the Inhibitor Cannons have the strange ability to slow down any ship they hit. Ships that we tested them on were unable to turn and accelerate at their normal rates."`

Reviewer 2: ```suggestion
			`	He then pulls out a schematic showing the weapon. "And finally, the weapon you brought us. The Inhibitor Cannon is quite the technical marvel, having properties we've never seen before. Imagine a Particle Cannon, but half the size and with more range. Two Inhibitor Cannons would take up about the same amount of space as a single Particle Cannon, yet the Inhibitor Cannons would deal more damage. On top of that, the projectiles of the Inhibitor Cannons have the strange ability to slow down any ship they hit. Ships that we tested them on were unable to turn and accelerate at their normal rates."`
```

			`	Ivan leans over to his desk. "This will be for your eyes only."`
				goto branches
			
			label again2
			`	"Again?" Ivan asks. He leans over to his desk. "I like that you're so interested in my research. I suppose it only makes sense with you helping me, but typically merchant captains aren't this interested in following up on odd jobs they do."`
			
			label branches
			branch one
				"ivan's suspicion" == 1
			
			branch two
				"ivan's suspicion" == 2
			
			branch three
				"ivan's suspicion" == 3
			
			branch four
				"ivan's suspicion" == 4
			
			label one
			apply
				set "remnant: got deep research"
			`	He presses a button on the desk. "Abigail, could you please make a copy of the encrypted folders 'TAB-RP' and 'RTS-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
				goto reencrypt
			
			label two
			apply
				set "remnant: got deep research"
			`	He hesistates for a moment before pressing a button on the desk. "Abigail, could you please make a copy of the encrypted folders 'TAB-RP' and 'RTS-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you.`

> 			`	"So, not handing this off to anyone, are we," he asks you.`

Reviewer 2: ```suggestion
			`	"So, not handing this off to anyone, are we?" he asks you.`
```

					goto course
				`	"No. Why do you ask?"`
			`	"Just curious," he responds.`
				goto reencrypt
			label course
			`	"Good," he responds.`
				goto reencrypt
			
			label three
			apply
				set "remnant: got old deep research"
			`	He hesistates for a moment, looks up at you, then presses a button on the desk while still looking into your eyes. "Abigail, could you please make a copy of the encrypted folders... 'TAI-RP' on the Institute network under my name and bring them to my office in a data chip? Thank you."`
			`	"So, you said you knew of the Remnant before working with me, right?" he asks you.`
			choice
				`	"Yeah. Why do you ask?"`
				`	"Not that I recall."`
					goto recall
			`	"Just curious. Do you care to tell me what your relationship with them is like?"`
				goto relationship
			label recall
			`	"Well I seem to recall that being what you told me. Do you care to tell me what your relationship with them is like?"`
			label relationship
			choice
				`	"I just do odd jobs for them. Nothing official."`
				`	"I'd rather not disclose that."`
			`	"Interesting," he says.`
				goto reencrypt
			
			label four
			apply
				set "remnant: got no deep research"

> 			`	He is about to press a button on his desk, then pulls away from it and turns his chair toward the courtyard. "Nevermind. I can't give you a copy. That research is classified."`

Reviewer 2: ```suggestion
			`	He is about to press a button on his desk, then pulls away from it and turns his chair toward the courtyard. "Never mind. I can't give you a copy. That research is classified."`
```

			choice
				`	"You've got me. I'll leave."`
					goto out
				`	"You have it all wrong, Ivan."`
			`	"I said get out of my office. Or do I need to call security?" Ivan hovers his finger over the button on the desk.`
			choice
				`	"Okay, okay. I'll leave."`
					goto out
				`	"I think you do need to call security. I'm not leaving without that reserach."`
			`	Ivan presses down on the button. "Security. I have a guest in my office who is no longer welcome."`
			`	Not even thirty seconds later, a group of security guards enter the office. "<first> <last>, you're going to need to come with us," one of them says.`
			choice
				`	"Okay, okay. I'll leave."`
					goto escorted
				`	"I don't think so."`
					goto resist
			
			label resist
			`	"Well I think so," the security guard quips back, pulling a taser from his holster. The other security guards equip various equipment of their own, some of it looking lethal. "I'm going to give you one more chance to leave."`
			choice
				`	"Okay, okay. I'll leave."`
					goto escorted
				`	"Well you're going to need to make me leave."`
			

> 			`	Before you can even finish your sentence, you can already feel your muscle tensing up from the taser. You collapse, hitting your head on Ivan's desk as you go down. Before you lose conciousness from the impact, you hear Ivan yell, "Come on! Now I'm going to need to get a new desk."`

Reviewer 2: ```suggestion
			`	Before you can even finish your sentence, you can already feel your muscles tensing up from the taser. You collapse, hitting your head on Ivan's desk as you go down. Before you lose consciousness from the impact, you hear Ivan yell, "Come on! Now I'm going to need to get a new desk."`
```

			`	"Aside from fleet movements, the number of ships was of note. The cubes track the number of unique ships seen, and they rarely tracked the same ship more than a handful of times. The Remnant seem to have an incredible number of ships, more than perhaps Deep Security and the Navy combined. Certainly impressive for a group consisting of only three star systems. The cubes are also able to make low quality scans of any ships in the system. While those individual scans are less than what an outfit scan provides, given enough scans you're able to get a clearer picture. These ship scan results were... interesting, though. There seemed to be very few ships like the ones you scanned. Perhaps those were their most advanced ships, but the rest of what I was able to see didn't seem more powerful than your average merchant ship."`

> 			`	"I actually lost connection to the cubes a few weeks ago now, but I've gathered enough information already."`

Reviewer 2: ```suggestion
			`	"I actually lost connection to the cubes a few weeks ago now, but I'd gathered enough information already."`
```

			`	"Do you think I could get a copy of your research?" you ask.`
			label "surveillance copy"
			`	"A copy? Of course! Of course! You're the reason I have any of this information in the first place, so you have every right to it." Ivan presses a button on his desk. "Abigail, could you please make a copy of everything on the Institute network under my name and bring them to my office in a data chip? Thank you."`
			apply
				set "remnant: got all deep research"
				
			label reencrypt

> 			`	Only a few minutes later there is a knock at the door. Ivan retrieves the data chip from the person who brought it, then plugs it into a computer on his desk. "Let me reencrypt these then leave the new key in the chip for you. Not exactly the most secure thing to do to leave the key right next to the lock, but I take it you'll be smart enough to remedy that whenever you use this. Aaaand... done." He pulls the chip from his computer and hands it to you.`

Reviewer 2: ```suggestion
			`	Only a few minutes later there is a knock at the door. Ivan retrieves the data chip from the person who brought it, then plugs it into a computer on his desk. "Let me reencrypt these then leave the new key in the chip for you. Leaving the key right next to the lock isn't exactly the most secure thing to do, but I take it you'll be smart enough to remedy that whenever you use this. Aaaand... done." He pulls the chip from his computer and hands it to you.`
```
Personal preference for flow.

			`	"Anything else?" he asks.`
			`	"No. That's all I was here for," you respond. "Good luck on the rest of your work."`

> 			`	"Thank you," Ivan says. "Come back any time you want. You're free to roam the Institute whenver you desire. Most of the building is closed to the public, but I can always mark you as my guest whenever you're interested in looking around."`

Reviewer 2: ```suggestion
			`	"Thank you," Ivan says. "Come back any time you want. You're free to roam the Institute whenever you desire. Most of the building is closed to the public, but I can always mark you as my guest whenever you're interested in looking around."`
```

mission "Remnant: Follow-up on Terminus Researchers A-1"
	name "Check up on Wormhole Researchers"
	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
	minor
	source
		government "Remnant"
	destination "Bounty"
	to offer
		has "Remnant: Learning Sign 2: done"
		has "Terminus exploration: done"
		not "Deep: Remnant 0: offered"
		random < 40
	on offer
		conversation
			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you first got your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants.`
			branch secrets
				has "deep: did not reveal remnant"

> 			`	Thinking back, it suddenly occurs to you that you helped retrieve a science drone for some researchers on Bounty - researchers who were studying the wormhole in Terminus. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes. It would only be a matter of time until someone figures out how to use the key stones either by luck or by research, and then the Remnant's secrecy would vanish.`

Reviewer 4: Everywhere else I've seen them referenced, they've been called **Keystones** .

Author: There are two separate but related outfits. The primary ones - the ones the Remnant have, who actually know what they are and how to use them, are called Quantum Key Stones. Other species who have similarly functioning stones don't know their true purpose, and thus the name has - over time - degenerated to Keystones. So when talking about the stones in general, they use the true name for these things.

Reviewer 4: What about capitalization then?

			`	"Greetings Captain; I am Prefect Torza. I was told you had some information that could be important?" He takes a seat on a bench on the far side of the room from the other Remnant and gestures for you to do the same.`
			branch confirmation
				has "deep: did not reveal remnant"
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system. He quickly picks up on the potential threat if someone figures out the key to using them, especially since you have demonstrated that it only takes one curious human to figure it out.`
				goto investigatechoice
			label confirmation

> 			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers who were using drones to study the wormhole exit in the Terminus system and now know that quantum keystones can stabilize it. He quickly picks up on the potential threat that this development represents.`

Reviewer 4: **Quantum Keystones**

Author: Here, on the other hand, they refer to the keystones because they are talking specifically about using the Hai keystones to stabilize it, not Key Stones as an overall concept.

			choice
				`	(Stop investigating.)`
					decline
				`	(Continue the investigation)`
					accept
	on complete
		conversation
			`Once you've landed on <planet>, you look up the local university and pay it a visit. After thinking about it for a minute, you decide that the best source of information would probably be to visit the university library and ask one of the librarians for assistance locating research on wormholes. After some brief searching, they return with a handful of research papers published over the past decade. They proudly note that two of the more recent papers were published here in the Deep, and transfer copies of each to you for free.`

mission "Remnant: Follow-up on Terminus Researchers AB-3"
	name "Check up on Terminus Researchers"
	description "Returning with information on the wormhole researchers."
	source
		government "Deep"
	destination
		government "Remnant"
	to offer
		or
			has "Remnant: Follow-up on Terminus Researchers A-2: done"
			has "Remnant: Follow-up on Terminus Researchers B: done"
	on offer
		conversation

> 			`Back on board <ship>, you take a moment to flip through the papers. You quickly realize that little beyond the abstract is understandable to someone without a good theoretical understanding of physics. Even with your first-hand experience of wormholes, most of what they are discussing doesn't make much sense.`

Reviewer 4: Do they still use physical paper?  The previous part makes it sound like a digital transfer (can't really flip through digital media).

Author: It could be actual papers; but that being said, I flip through pdf's all the time, even if there is no actual paper.

Could be something as simple as being an old term that has stuck with humanity, much like present day operating systems frequently use a little disk icon to represent "save", despite the fact that we have large swaths of computer-using people who have never even seen a floppy disk, let alone used one regularly as a means of saving work.
